checksyntax/check_syntax.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+# Path to this source file
+export parent_directory=$(dirname "$(pwd)")
+export check_directory="$parent_directory/checksyntax/.checksyntax"
+
+process_inc_file() {
+  echo "Checking $1"
+  output_file="$check_directory/${1##*/}.gcc"
```

Comment:
Can you explain what this command is doing?

Reply:
The file can be something like './src/utility/utils.inc'. This command will set the name of the output file to "$check_directory/utils.inc.gcc"